dist/tilegrid.js

```diff
@@ -1,10 +1,10 @@
 (function webpackUniversalModuleDefinition(root, factory) {
 	if(typeof exports === 'object' && typeof module === 'object')
-		module.exports = factory(require("_"), require("jquery"), require("backbone"), require("react"), require("react-dom"));
```

Comment:
Not sure why these change while I grunt build. These cause errors. @littlebee any idea why this is happening?